src/appdistribution/distribution.ts

```diff
@@ -49,13 +58,18 @@ export class Distribution {
     switch (this.fileType) {
       case DistributionFileType.IPA:
         return "ios";
+      case DistributionFileType.AAB:
       case DistributionFileType.APK:
         return "android";
       default:
         throw new FirebaseError("Unsupported distribution file format, should be .ipa or .apk");
```

Comment:
Update to "unsupported distribution file format, should be .ipa, .apk or .aab"

Reply:
Done!